### ISSUE TITLE
OTP Verification is throwing exception

### ISSUE THREAD

**Issue by dhruva81** [craftsys/msg91-laravel-notification-channel#7]
Hi 

Earlier OTP verification was working fine. 
But now I'm getting following exception when trying to verify the OTP.

```
 'Required parameter $errors follows optional parameter $message' .
```

My application is built on laravel 8.
craftsys/msg91-laravel : 0.13.0,
craftsys/msg91-laravel-notification-channel: 0.6.0,

Kindly help.

**Comment by sudkumar** [comment]
Hi @dhruva81, Thanks for reporting. I will look into it.

If possible, please provide a minimal reproduction source code.

> Avoid adding your MSG91 Auth Keys in any public source code. If needed, I will request you more details in a private email conversation.

**Comment by sudkumar** [comment]
@dhruva81 Can you please confirm if you are on PHP 8.0 ?

**Comment by dhruva81** [comment]
Hi 
Thank you for reply. 
I am using PHP 8.

**Comment by sudkumar** [comment]
@dhruva81  Please update to latest version `0.6.1` and test it out. Should be fixed.

Here is the change log: https://github.com/craftsys/msg91-php/pull/8

**Comment by dhruva81** [comment]
@sudkumar 

It is now working. Thank you very much :100:  :+1:

**Comment by sudkumar** [comment]
Thank you for the confirmation. Closing it now.

Fixed in [0.6.1](https://github.com/craftsys/msg91-laravel-notification-channel/releases/tag/v0.6.1)
